plonetheme/onegovbear/theme/scss/grid.scss

```diff
@@ -1,71 +0,0 @@
-$column-width: 56px;
-$gutter-width: 18px;
-$margin-width: 18px;
-$columns: 16;
-
-$gridsystem-width: $columns * $column-width + ($columns - 1) * $gutter-width + 2 * $margin-width;
-
-@for $x from 1 through $columns {
```

Comment:
Moved the grid system to ftw.theming and kept the grid classes because bern.web is not working without them  -> https://github.com/4teamwork/ftw.theming/commit/a4220c3266c24d33b082f7ac5d65c6bf558a04bb